plugins/jira-dashboard-backend/src/service/router.ts

```diff
@@ -215,6 +217,42 @@ export async function createRouter(
     },
   );
 
+  router.get('/dashboards/userIssues', async (request, response) => {
```

Comment:
Suggestion: "user-issues" instead with kebab case? Since the other routes look like: '/dashboards/by-entity-ref/:kind/:namespace/:name'.